CommonMark/Syntax/Inline.cs

```diff
@@ -205,5 +206,14 @@ public Inline LastSibling
                 return x;
             }
         }
+
+        /// <summary>
+        /// Gets the delimiter character for this inline element.
+        /// </summary>
+        public char DelimiterCharacter
+        {
+            get;
+            private set;
```

Comment:
I just learned that you cannot apply `ObsoleteAttribute` to a property accessor. That's a shame.